jerry-ext/debugger/debugger-tcp.c

```diff
@@ -167,7 +182,10 @@ jerryx_debugger_tcp_send (jerry_debugger_transport_header_t *header_p, /**< tcp
     }
 #endif /* __linux__ */
 
-    ssize_t sent_bytes = send (tcp_p->tcp_socket, message_p, message_length, 0);
+    jerryx_socket_ssize_t sent_bytes = send (tcp_p->tcp_socket,
+                                              (const jerryx_socket_void_t *) message_p,
```

Comment:
Let me write it again, drop the `const` qualifier from the cast. The `const` qualifier in the signature of `send` is a guarantee that it will not modify the contents of the buffer, not a requirement to the caller to pass in an unmodifiable buffer.

Reply:
as you insist, then i remove the const,
but consider this 
if people modify the message_p parameter from uint8_t * to const uint8_t *,
then this cast will hidden the compiling error, doing this just hidding things

Reply:
If you rewrite the signature of the function to take `message_p` as a pointer to const (and/or rewrite the opt value to be a `const int`) then I'll accept a const cast.